deafrica/data/gmw.py

```diff
@@ -73,10 +74,8 @@ def create_and_upload_stac(cog_file: Path, s3_dst: str, year) -> Item:
     )
 
     log.info("assets creation")
-    del item.assets["asset"]
-
     out_data = out_path / cog_file.name
-    item.assets["mangrove "] = pystac.Asset(
+    item.assets["asset"] = pystac.Asset(
```

Comment:
I think you should be doing this using the arguments for the `create_stac_item` function

Reply:
I am doing that:
https://github.com/digitalearthafrica/deafrica-scripts/blob/518d9457508917aaf42a84d50916b2be2ad6ec33/deafrica/data/gmw.py#L50

Reply:
I think I did it the wrong way.

Pass in `asset_name="mangrove"` and `asset_roles=["data"]` and `asset_href=...`

Reply:
This was the way that I was doing before and you asked me to change lol..... I'll roll back.